### DISCUSSION TITLE
ZB Bridge: Raw message

### DISCUSSION THREAD

**Discussion by alsak0de** [arendst/Tasmota#16755] [Q&A]
Hiya, from zigbee bridge console:

10:41:25.142 ZIG: {"ZbEZSPReceived":"2400FA192A9082FEFF14430C02039FC2"}

How do I find out what device corresponds to? does that mean it has not been properly binded to the right cluster?
Any clue is appreciated.
Thanks in advance,

**Comment by s-hadinger** [comment]
Those are debug messages between ezsp MCU and Tasmota. You don't need to bother. What are the logs before and after?

**Comment by alsak0de** [comment]
Ok, thanks! Logs and operation are just fine.
I was just curious, trying to find 'lost' devices.
